### PR TITLE
zib-FLACCPainScale

### PR DESCRIPTION
**Pull request by furoreSebastiaan**

### CODE REVIEW

File: Profiles - ZIB 2017/zib-FLACCPainScale.xml [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+      <path value="Observation.component" />
+      <sliceName value="Face" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:Face.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org" />
+          <code value="MISSING" />
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/zib-FLACCPainScale.xml]: Code is still MISSING

File: Profiles - ZIB 2017/zib-FLACCPainScale.xml [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+      <path value="Observation.component" />
+      <sliceName value="Legs" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:Legs.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org" />
+          <code value="MISSING" />
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/zib-FLACCPainScale.xml]: Code is still MISSING

Comment (furoreSebastiaan) [reply Profiles - ZIB 2017/zib-FLACCPainScale.xml]: fair point!

File: Profiles - ZIB 2017/zib-FLACCPainScale.xml [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+    <element id="Observation.code.coding:FLACCPainScaleCoding.system">
+      <path value="Observation.code.coding.system" />
+      <min value="1" />
+      <fixedUri value="http://loinc.org" />
+    </element>
+    <element id="Observation.code.coding:FLACCPainScaleCoding.code">
+      <path value="Observation.code.coding.code" />
+      <min value="1" />
+      <fixedCode value="38213-5" />
+    </element>
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/zib-FLACCPainScale.xml]: Maybe add the default display?

Comment (furoreSebastiaan) [reply Profiles - ZIB 2017/zib-FLACCPainScale.xml]: added display defaults, please review

**Comment by furoreSebastiaan** [comment]
please review!

File: Profiles - ZIB 2017/zib-FLACCPainScale.xml [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+        <map value="NL-CM:12.13.1" />
+        <comment value="FLACCpainScore" />
+      </mapping>
+    </element>
+    <element id="Observation.code.coding">
+      <path value="Observation.code.coding" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="system" />
```
Comment (ArdonToonstra) [inline Profiles - ZIB 2017/zib-FLACCPainScale.xml]: Surely on discrimination on system and code? double-check

File: Profiles - ZIB 2017/zib-FLACCPainScale.xml [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="hcim-flaccpainscale-v1.0-2017EN" />
+        <map value="NL-CM:12.13.11" />
+        <comment value="ScoreDateTime" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]:valueQuantity">
+      <path value="Observation.valueQuantity" />
```
Comment (ArdonToonstra) [inline Profiles - ZIB 2017/zib-FLACCPainScale.xml]: Missing zib alias, shortname, definition ect?

File: Examples/zib-FLACCPainScore - example.xml [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+            <display value="FLACC pain assessment panel" />
+        </coding>
+    </code>
+    <subject>
+        <reference value="Patient/example"/>
+        <display value="Example Patient"/>
+    </subject>
+    <effectiveDateTime value="2013-02-08T06:43:00+02:00"/>
+    <valueQuantity>
+        <value value="6"/>
```
Comment (ArdonToonstra) [inline Examples/zib-FLACCPainScore - example.xml]: provide a more enhanced valueQuantity if possible

File: Profiles - ZIB 2017/zib-FLACCPainScale.xml [hunk 7]
```diff
@@ -0,0 +1,353 @@
...
+      <alias value="Gezicht" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <description value="GezichtCodelijst" />
+        <valueSetReference>
+          <reference value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.12.13.1--20171231000000" />
+        </valueSetReference>
```
Comment (ArdonToonstra) [inline Profiles - ZIB 2017/zib-FLACCPainScale.xml]: missing display of valueset

File: Profiles - ZIB 2017/zib-FLACCPainScale.xml [hunk 8]
```diff
@@ -0,0 +1,353 @@
...
+      <alias value="Benen" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <description value="BenenCodelijst" />
+        <valueSetReference>
+          <reference value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.12.13.2--20171231000000" />
+        </valueSetReference>
```
Comment (ArdonToonstra) [inline Profiles - ZIB 2017/zib-FLACCPainScale.xml]: missing valueset reference

File: Profiles - ZIB 2017/zib-FLACCPainScale.xml [hunk 9]
```diff
@@ -0,0 +1,353 @@
...
+      <definition value="Observation activity: the extent to which the child is restless and has tense muscles." />
+      <alias value="Activiteit" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <description value="ActiviteitCodelijst" />
+        <valueSetReference>
+          <reference value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.12.13.3--20171231000000" />
```
Comment (ArdonToonstra) [inline Profiles - ZIB 2017/zib-FLACCPainScale.xml]: Same as the earlier two comments, please add them. It is just a minor improvement, but a nice one :)

File: Examples/zib-FLACCPainScore - example.xml (hunk 6 above)
Comment (ArdonToonstra) [reply Examples/zib-FLACCPainScore - example.xml]: Ignore my feedback.

**Pull request merged**